### PR TITLE
Return correct nullable value in information_schema.columns

### LINKED ISSUE #6400: Actual isNullable values not populated in system.jdbc.columns/hive.informationschema.columns/<jdbcdb>.informationschema.columns
isNullable seems to always show 2 (ie UNKNOWN). Regardless of whether I create a table with/without columns set to NOT NULL or actually containing null values in columns. select distinct nullable from system.jdbc.columns 
where table_cat = 'system' also returns just 2 

nullable

To have an accurate data dictionary it should match the isNullable value from the underlying db (ie sqlserver/postgres.etc or hive metastore since v3)

https://github.com/prestosql/presto/blob/348/presto-main/src/main/java/io/prestosql/connector/system/jdbc/ColumnJdbcTable.java#L311

https://github.com/prestosql/presto/blob/348/presto-spi/src/main/java/io/prestosql/spi/connector/ColumnMetadata.java#L43

https://github.com/prestosql/presto/blob/348/presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java#L293

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Return correct nullable value in information_schema.columns
Fixes #6400

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
(x) Release notes entries required with the following suggested text:

```markdown
# General
* Fix incorrect `is_nullable` value in `information_schema.columns` and `system.jdbc.columns` tables. ({issue}`11613`)
```

### CODE REVIEW

**Review by hashhar — APPROVED** [review]
Thank you.

File: core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaPageSource.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ private void addColumnsRecords(QualifiedTablePrefix prefix)
                         column.getName(),
                         ordinalPosition,
                         null,
-                        "YES",
+                        column.isNullable() ? "YES" : "NO",
```
Comment (findepi) [inline core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaPageSource.java:266]: please update `system.jdbc.columns` too
https://github.com/trinodb/trino/blob/c0276cea688da155f7f5bf0cb0b53169ece43e6e/core/trino-main/src/main/java/io/trino/connector/system/jdbc/ColumnJdbcTable.java#L329

**Comment by ebyhr** [comment]
Fixing CI failure.

**Review by findepi — APPROVED** [review]

**Comment by ebyhr** [comment]
(Pushed an empty commit as Hive PT wasn't triggered)

**Comment by ebyhr** [comment]
CI #11054

**Pull request merged**